Documentation/using-rkt-with-systemd.md

```diff
@@ -115,7 +118,9 @@ Requires=network-online.target
 After=network-online.target
 
 [Service]
+Slice=machine.slice
 # Resource limits
+Delegate=true
```

Comment:
Following #1883...

I kept Delegate=true here because it does not cost anything and it is similar to the nspawn@ service, but I removed `-p delegate` from the systemd-run command line because it is cumbersome to the user and does not bring anything.